client/components/main/layouts.js

```diff
@@ -2,10 +2,52 @@ Meteor.subscribe('boards');
 
 BlazeLayout.setRoot('body');
 
+// see: https://github.com/softwarerero/meteor-accounts-t9n#language-codes-and-contributions
+const mapTagi18nToT9n = {
+  'es-ES': 'es_ES',
+  'no-NB': 'no_NB',
+  'pt-BR': 'pt',
+  'pt-PT': 'pt_PT',
+  'zh-CN': 'zh_cn',
+  'zh-HK': 'zh_hk',
+  'zh-TW': 'zh_tw',
+};
+
+const i18nTagToT9n = (i18nTag) => {
```

Comment:
Since https://github.com/wekan/wekan/pull/421 was released, we can now remove this.